### PR TITLE
clean up parameter html table

### PR DESCRIPTION
**Pull request by newville**
This cleans up #552

### CODE REVIEW

**Comment by newville** [comment]
@reneeotten I think this is uncontroversial.  Any objections to merging?

File: lmfit/printfuncs.py [hunk 1]
```diff
@@ -278,29 +278,29 @@ def cell(x, cat='td'):
...
+    for par in params.values():
+        rows = [par.name, gformat(par.value)]
         if has_err:
-            serr, perr = '', ''
-            if p.stderr is not None:
-                serr = gformat(p.stderr)
+            serr = ''
+            if par.stderr is not None:
+                serr = gformat(par.stderr)
             try:
```
Comment (reneeotten) [inline lmfit/printfuncs.py]: shouldn't the indentation be at the same position as the line above, similar to what's done in the ```fit_report``` function? (https://github.com/lmfit/lmfit-py/blob/e465b8269aaa447d10b60e8139cc24ddb1259757/lmfit/printfuncs.py#L174-L179)

Comment (newville) [reply lmfit/printfuncs.py]: yes, you are right... will fix.

**Comment by reneeotten** [comment]
My two cents on this: I don't think we should merge PRs and then follow-up with a clean-up PR... but I guess you already expected that ;) I asked for the same changes in that other PR (which had 5 (redundant) commits, didn't change the code in the way we wanted plus it failed the test-suite). The whole idea of PR is that one get comments on the proposed changes and that code gets merged once everyone is satisfied.

**Comment by newville** [comment]
@reneeotten  I do understand and respect your concerns and am sort of sorry to do this.  But I also sort of felt like it would be nice to encourage and acknowledge new contributors.  

I've experienced too many projects  *not* merging PRs and so actually missing contributions and pushing away contributors.  I would rather accept a gift that needs a repair than reject it and lose not only the gift but the friend.

I do completely admit that I'm not very good at keeping really clean Git workflows.  I'm trying but I seem to mess it up more often than not.

**Comment by reneeotten** [comment]
@newville I do understand your point as well and am totally in favor of encouraging/acknowledging new contributors. And yes, we should certainly these merge PRs, but I also think there is nothing wrong with asking a (new) contributor to gift-wrap their present nicely ;)

**Pull request merged**